### PR TITLE
feat(cb2-6425): Adds formatting and styling to all VRMs/number plates

### PR DESCRIPTION
**Pull request by andrewf-bjss**
Adds formatting and styling to all VRMs/number plates

https://dvsa.atlassian.net/browse/CB2-6425

## Checklist

- [x] Branch is rebased against the latest develop/common
- [x] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [x] Code and UI has been tested manually after the additional changes
- [x] PR title includes the JIRA ticket number
- [x] Squashed commits contain the JIRA ticket number
- [x] Link to the PR added to the repo
- [x] Delete branch after merge

### CODE REVIEW

**Review by Ruairidh-BJSS — DISMISSED** [review]

**Comment by andrewf-bjss** [comment]
merged in latest `develop` and fixed merge conflict

File: src/app/shared/components/number-plate/number-plate.component.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<span class="govuk-number-plate">
-  <ng-content></ng-content>
+<span class="govuk-number-plate" [class.secondary]="isSecondary">
```
Comment (gjulien-bjss) [inline src/app/shared/components/number-plate/number-plate.component.html:1]: I think `ngClass` might be the preferred way of doing this
```suggestion
<span class="govuk-number-plate" [ngClass]="{'secondary': isSecondary}">
```

File: src/app/shared/components/number-plate/number-plate.component.ts [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class NumberPlateComponent {}
+export class NumberPlateComponent {
+  private _vrm: string | undefined;
+
+  @Input() isSecondary: boolean = false;
+  @Input() set vrm(value: string | undefined) {
+    // formatting: if the number plate is long enough, add a space before the final 3 characters
+    if (value && value.length >= 5) {
```
Comment (gjulien-bjss) [inline src/app/shared/components/number-plate/number-plate.component.ts:15]:
```suggestion
    if (value?.length >= 5) {
```

File: src/app/shared/components/number-plate/number-plate.component.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
+  {{ vrm }}
```
Comment (gjulien-bjss) [inline src/app/shared/components/number-plate/number-plate.component.html:2]:
```suggestion
  {{ vrm | defaultNullOrEmpty}}
```

**Review by gjulien-bjss — APPROVED** [review]

**Review by mcadenas-bjss — DISMISSED** [review]

**Review by gjulien-bjss — APPROVED** [review]

**Pull request merged**